doc/editorconfig.txt

```diff
@@ -189,10 +189,28 @@ file:
 
      return 0   " Return 0 to show no error happened
  endfunction
-
+<
+If the plugin is installed with a plugin manager that loads it while parsing
+the |vimrc|, then we can immediately add this hook:
+>
  call editorconfig#AddNewHook(function('FiletypeHook'))
 <
-And add the following code to your .editorconfig file:
+If the plugin is installed as part of a Vim 8 |package|, it will not be loaded
+until after the |vimrc| file has been parsed. This may be the case with some
+other plugin managers too. As a result, the above call to add the hook will not
+work as the function will not have been loaded yet. Instead, we can use the
+|SourcePost| autocommand event to add the hook after the plugin is loaded:
+>
+ autocmd SourcePost */plugin/editorconfig.vim call editorconfig#AddNewHook(function('FiletypeHook'))
```

Comment:
This looks reasonable to me.  However, could you use the `VimEnter` event instead?  That way it would not depend on the filename.  I don't think the filename will ever change, but I'd rather not add that restriction if we don't have to.

I tried this just now and it worked (vim 8.1, Ubuntu package ver. 2:8.1.2269-1ubuntu5):
```vim
let g:CrazyCount = 0
function MyCrazyHook(config)
  echom "Crazy!"
  let g:CrazyCount = g:CrazyCount+1
  return 0
endfunction

let g:EditorConfig_verbose = 1

augroup ectest
  autocmd!
  autocmd VimEnter * call editorconfig#AddNewHook(function('MyCrazyHook'))
augroup END
```